view-in-grafana.sh

```diff
@@ -1,59 +1,114 @@
 #!/bin/bash
 
+# VARIABLES
+BUILDLOCAL=false
+INFLUXDB=false
```

Comment:
I thought influxdb was going to be the default and graphite was going to be opt-in?

Reply:
I added a new commit to default to InfluxDB.